root/admin/delete_user.tt

```diff
@@ -8,11 +9,7 @@
         <button class="negative" type="submit">[% l('Delete {e}', { e => user.name } ) %]</button>
       </span>
     </form>
-    [% END %]
 [% ELSE %]
-    [% WRAPPER 'layout.tt' title=l('Delete Account') full_width=1 %]
-    <h1>[% l('Delete Account') %]</h1>
-
```

Comment:
I have absolutely no clue why this was a) inside the IF and b) different, but I can't see any good reasons, so generalising it.